src/pages/[guild]/events.tsx

```diff
@@ -51,14 +52,15 @@ const GuildEvents = (): JSX.Element => {
 
   const discordGuildPlatform = guildPlatforms?.find(
     (platform) => platform.platformId === PlatformType.DISCORD
-  )
+  )?.platformGuildId
 
-  const { data, isLoading, error } = useDiscordEvents(
-    discordGuildPlatform?.platformGuildId
+  const { data, isLoading, error, serverError } = useGuildEvents(
+    discordGuildPlatform,
```

Comment:
Consider using the `useGuild` hook inside `useGuildEvents` - that way you wouldn't need to pass these props